--- lxmenu-data-0.1.2/configure.in~	2011-07-23 22:05:34.000000000 +0200
+++ lxmenu-data-0.1.2/configure.in	2014-04-30 18:20:09.985465810 +0200
@@ -6,7 +6,10 @@
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [Name of default gettext domain])
 AC_SUBST(GETTEXT_PACKAGE)
 
-AM_GLIB_GNU_GETTEXT
+#AM_GLIB_GNU_GETTEXT
+#http://blogs.gnome.org/jjardon/2010/10/08/use-upstream-gettext-instead-the-glib-one/
+AM_GNU_GETTEXT([external])
+AM_GNU_GETTEXT_VERSION([0.17])
 
 IT_PROG_INTLTOOL([0.40.0])
 
